### DISCUSSION TITLE
Cannot read properties of undefined (reading 'bitmap')

### DISCUSSION THREAD

**Discussion by ghost** [gatsbyjs/gatsby#36245] [Help] [answered]
I am a gatsby beginner. Follow the youtube tutorial to build a website with gatsby, and now I am planning to deploy it with netlify. But netlify says can't deploy.
I have run "gatsby develop" and "gatsby build", they are no problem.

Reproduction Link:
https://github.com/dealday2013/gatsby

10:01:51 AM: error Cannot read properties of undefined (reading 'bitmap')
10:01:51 AM: 
10:01:51 AM: 
10:01:51 AM:   TypeError: Cannot read properties of undefined (reading 'bitmap')
10:01:51 AM:   
10:01:51 AM:   - Potrace.js:1002 Potrace._processLoadedImage
10:01:51 AM:     [repo]/[@gatsbyjs]/potrace/lib/Potrace.js:1002:35
10:01:51 AM:   
10:01:51 AM:   - Potrace.js:1048 Jimp.<anonymous>
10:01:51 AM:     [repo]/[@gatsbyjs]/potrace/lib/Potrace.js:1048:14
10:01:51 AM:   
10:01:51 AM:   - index.js:223 Timeout._onTimeout
10:01:51 AM:     [repo]/[@jimp]/core/src/index.js:223:9
10:01:51 AM:   
10:01:51 AM:   - timers:559 listOnTimeout
10:01:51 AM:     node:internal/timers:559:17
10:01:51 AM:   
10:01:51 AM:   - timers:502 processTimers
10:01:51 AM:     node:internal/timers:502:7
10:01:51 AM:   
10:01:51 AM: 
10:01:51 AM: error An error occurred during parallel query running.
10:01:51 AM: Go here for troubleshooting tips: https://gatsby.dev/pqr-feedback
10:01:52 AM: 
10:01:52 AM: 
10:01:52 AM:   Error: Worker exited before finishing task
10:01:52 AM:   
10:01:52 AM:   - index.js:117 ChildProcess.<anonymous>
10:01:52 AM:     [repo]/[gatsby-worker]/dist/index.js:117:45
10:01:52 AM:   
10:01:52 AM:   - node:events:527 ChildProcess.emit
10:01:52 AM:     node:events:527:28
10:01:52 AM:   
10:01:52 AM:   - child_process:291 Process.ChildProcess._handle.onexit
10:01:52 AM:     node:internal/child_process:291:12
10:01:52 AM:   
10:01:52 AM: 
10:01:52 AM: not finished Running gatsby-plugin-sharp.IMAGE_PROCESSING jobs - 47.339s
10:01:52 AM: not finished run queries in workers - 3.929s
10:01:53 AM: ​
10:01:53 AM: ────────────────────────────────────────────────────────────────
10:01:53 AM:   "build.command" failed                                        
10:01:53 AM: ────────────────────────────────────────────────────────────────
10:01:53 AM: ​
10:01:53 AM:   Error message
10:01:53 AM:   Command failed with exit code 1: npm run build (https://ntl.fyi/exit-code-1)
10:01:53 AM: ​
10:01:53 AM:   Error location
10:01:53 AM:   In Build command from Netlify app:
10:01:53 AM:   npm run build
10:01:53 AM: ​
10:01:53 AM:   Resolved config
10:01:53 AM:   build:
10:01:53 AM:     command: npm run build
10:01:53 AM:     commandOrigin: ui
10:01:53 AM:     environment:
10:01:53 AM:       - CONTENTFUL_API_KEY
10:01:53 AM:     publish: /opt/build/repo/public
10:01:53 AM:     publishOrigin: ui
10:01:53 AM:   plugins:
10:01:53 AM:     - inputs: {}
10:01:53 AM:       origin: ui
10:01:53 AM:       package: '@netlify/plugin-gatsby'
10:01:53 AM: Caching artifacts
10:01:53 AM: Started saving node modules
10:01:53 AM: Finished saving node modules
10:01:53 AM: Started saving build plugins
10:01:53 AM: Finished saving build plugins
10:01:53 AM: Started saving pip cache
10:01:53 AM: Finished saving pip cache
10:01:53 AM: Started saving emacs cask dependencies
10:01:53 AM: Finished saving emacs cask dependencies
10:01:53 AM: Started saving maven dependencies
10:01:53 AM: Finished saving maven dependencies
10:01:53 AM: Started saving boot dependencies
10:01:53 AM: Finished saving boot dependencies
10:01:53 AM: Started saving rust rustup cache
10:01:53 AM: Finished saving rust rustup cache
10:01:53 AM: Started saving go dependencies
10:01:53 AM: Finished saving go dependencies
10:01:55 AM: Build failed due to a user error: Build script returned non-zero exit code: 2
10:01:55 AM: Creating deploy upload records
10:01:55 AM: Failing build: Failed to build site
10:01:55 AM: Failed during stage 'building site': Build script returned non-zero exit code: 2 (https://ntl.fyi/exit-code-2)
10:01:55 AM: Finished processing build request in 2m31.795999297s

**Comment by jongwooo** [comment, marked as answer]
@dealday2013 On your [queries](https://github.com/dealday2013/gatsby/blob/main/src/components/AllRecipes.js), remove `TRACED_SVG` from the images. I hope this helps!

**Reply by ghost** [reply]
@jongwooo thank you very much. Change the pictures to `BLURRED` and it will be solved. By the way, can `TRACED_SVG` not be available in netlify in the future?

**Reply by jongwooo** [reply]
@dealday2013 I don't know if netlify is aware of this, but hopefully it will be resolved :)

**Reply by jongwooo** [reply]
@dealday2013 Can you mark my comment as an answer?

**Reply by ghost** [reply]
@jongwooo yes, but I don't know how to mark.., I can't find the button..

**Reply by ghost** [reply]
I find it now, thank you very much
